pkg/asset/manifests/kube-core-operator.go

```diff
@@ -63,7 +63,7 @@ func (kco *kubeCoreOperator) Generate(dependencies map[asset.Asset]*asset.State)
 	state := &asset.State{
 		Contents: []asset.Content{
 			{
-				Name: "kube-core-operator-config.yml",
+				Name: "kco-config.yaml",
```

Comment:
nit: I'm fine with `yml` -> `yaml`, but I like `kube-core-operator-` more than `kco-`.

Reply:
This is to match the existing output. We can rename these afterward.